scripts/crab/commentOutEmptyNtuples.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#####################
+##  DOCUMENTATION  ##
+#####################
+
+# Empty Ntuples may cause sframe_main to crash under certain circumstances, e.g. if the last Ntuple in the
+# list of input files within the job's xml file is an empty one. Thus, you can call this script to comment out
+# empty Ntuples in the dataset xml in the first place (creates a new dataset xml, old one is not overwritten).
+
+# This sframe_main bug should be fixed at some point within the actual sframe code but, for now, this is a nice workaround.
+
+# After your CRAB jobs are finished and you created the XML, call this script just like you do with readaMCatNloEntries.py
+
+# Usage:
+# >>> python commentOutEmptyNtuples.py ../../common/datasets/DATA_SingleElectron_RunB.xml
+# Creates: ../../common/datasets/DATA_SingleElectron_RunB_withoutEmptyNtuples.xml
+
+
+import sys
+import os
+
+def read_xml(xmlFileDir):
+    xmlFile = open(str(xmlFileDir))
+    rootFileStore = []
+    comment = False
```

Comment:
This bool doesn't seem to get used? Only set in the `if ` statements.

Reply:
See other message. Now, this boolean has a meaning.